docs/pages/machine-workload-identity/mwi-use-cases/mwi-ci-cd.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: CI/CD with Machine & Workload Identity
+description: Configuring CI/CD with Machine & Workload Identity
+labels:
+ - conceptual
+ - mwi
+---
+
+Teleport Machine & Workload Identity eliminates the need for long-lived static secrets in CI/CD pipelines by issuing short-lived, automatically generated certificates at runtime.
+Teleport natively integrates with many CI/CD providers and deployment targets. Refer to the links at the bottom of this page for instructions on using Machine & Workload Identity with your preferred tools.
+
+## Choose your CI/CD platform
+
+<TileGrid
+  tiles={[
+    {
+      icon: <Icon name="azureDevops" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/azure-devops",
+      name: "Azure DevOps",
+    },
+    {
+      icon: <Icon name="bitbucket" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/bitbucket",
+      name: "BitBucket",
+    },
+    {
+      icon: <Icon name="circleci" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/circleci",
+      name: "CircleCI",
+    },
+    {
```

Comment:
What about GitHub Actions? It's got to be one of the most popular options we offer. Should we include it here?